### DISCUSSION TITLE
Elementary OS 7.1 Installation error for Nvidia graphics cards and tty1 Elementary OS CLI login screen

### DISCUSSION THREAD

**Discussion by theofficialitbros** [elementary/triage#492] [Q&A]
I have tried multiple times now to install elementary os onto a partition of one of my 4 terabyte hard drives i've tried it in safe mode I have tried it in recovery mode i've tried it installing with the gui i've tried it installing with the cli and every time the installation completes but then when I try to boot into the os it automatically boots me to a cli interface where it asks me to log in to an elementary os tty1 login Which I do not have a user name or password for.

Please help I cannot seem to get past this.

https://github.com/elementary/triage/assets/150261875/ea2fb2e0-090d-488a-a3b7-d0c44ae90311

**Comment by jeremypw** [comment]
Have you tried pressing `<Control><Alt>F7`?  Pressing `<Control><Alt>F1` to `<Control><Alt>F6` should take you to `tty` to `tty6` but `Control><Alt>F7` should show the graphic interface (if it is running).
